epochStart/bootstrap/process.go

```diff
@@ -39,9 +39,12 @@ import (
 
 var log = logger.GetOrCreate("epochStart/bootstrap")
 
-const timeToWait = 10 * time.Second
+const timeToWait = time.Minute
+const trieSyncWaitTime = 5 * time.Minute
 const timeBetweenRequests = 100 * time.Millisecond
 const maxToRequest = 100
+const gracePeriodInPercentage = float64(0.25)
```

Comment:
This is equivalent to ShuffledOutRestartThreshold from config.toml [EpochStartConfig] section ?

Reply:
right now yes. but I do not know what gracePeriod should we choose.

Reply:
I think we should add a TODO here to not forget later to replace these constants, gracePeriodInPercentage and roundGracePeriod, with values read from config.toml

Reply:
I think this should not be configurable. we have to find the best values.